cloud/aws/benchmarks/benchmarks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+TEST_FLAGS="-test.bench=.* -test.benchmem -test.run=NONE -test.count=10"
```

Comment:
turning off `--benchmem` didn't save any time on a full run (all benchmarks with `--count=10`. Both took about 1h05m (without `benchmem` was actually 3 minutes longer, but that's noise)